### ISSUE TITLE
language-service does not seem to support tsconfig alias parameters

### ISSUE THREAD

**Issue by ClementVidal** [angular/angular#18484]
## I'm submitting a...
<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
</code></pre>

### Assuming the following directory layout:

```
.
|-- /src/
      |-- /app/
            |-- app.module.ts
            |-- app.routing.ts
            |-- /AppComponentA/
            |-- /AppComponentB/
      |-- /common/
            |-- common.module.ts
            |-- common.routing.ts
            |-- /CommonComponentA/
            |-- /CommonComponentB/
```

### And those two modules:

* AppModule
* CommonModule

### And knowing that:

Because i'm planning to isolate the common module in it's own git repo and npm package,
i'm actually referring to it using `@myns/common`.
Any reference in the AppModule to "anything" inside CommonModule is done using `@myns/common`.

`@myns/common` is an alias configured in:

**_tsconfig.json:_**
```
"baseUrl": ".",
"paths": {
    "@myns/common": ["src/common/"] // This mapping is relative to "baseUrl"
}
```
*and my webpack config:*
```
resolve: {
     alias: {
          '@myns/common': "./src/common",
     }
},
```

### AppModule import CommonModule the following way:

```
import { CommonModule } from '@myns/common';

@NgModule({
    imports: [
        CommonModule
    ]
})
export class AppModule {

}
```
**This works, compile and execute just fine.**

But, when using vscode version 1.14.2 and the Angular language service plugin version 0.1.4 (which seams to use angular language service 4.1.0-beta), every components from CommonModule and referenced in a template inside AppModule aren't recognized, and i get the following message

> `my-common-directive` is not a known element.

_**If i switch from importing the CommonModule in AppModule using the aliased name to the absolute path of the module, then everything work just fine.**_

_Exemple:_
If i change `import { CommonModule } from '@myns/common';`   
to `import { CommonModule } from '../common';` it work !

## Expected behavior
Language service should not report errors when using aliased imports.

## Environment

<pre><code>
Angular version: 4.3.2
 
For Tooling issues:
- Node version: 6.11.0
- Platform:  Linux

Others:
- VsCodde 1.14.2
- Angular language service plugin 0.1.4
- Angular language service 4.1.0-beta
</code></pre>

**Comment by ylibrach** [comment]
Related to #16382 (possibly also a duplicate).

**Comment by kyliau** [comment]
this should have already been fixed by #20222

**Comment by bkbaheti** [comment]
One solution for webstorm, is to disable the angular service, but this is more like killing the messenger. Any updates on the fix?

**Comment by WiseBird** [comment]
@bkbaheti, works ok for me in latest version.

**Comment by bkbaheti** [comment]
@WiseBird no luck, have updated everything

**Comment by stevo-knievo** [comment]
I have the same problem with IntelliJ IDEA 2017.3.5 but it works fine with VSCode 1.21.1
Angular language service 6.0.0-rc.0

**Comment by JoostK** [comment]
I can no longer reproduce in latest language service, so this has indeed likely been resolved in #20222.
